10.0/Dockerfile

```diff
@@ -39,7 +39,9 @@ RUN chown odoo /etc/odoo/odoo.conf
 
 # Mount /var/lib/odoo to allow restoring filestore and /mnt/extra-addons for users addons
 RUN mkdir -p /mnt/extra-addons \
-        && chown -R odoo /mnt/extra-addons
+    && chown -R odoo /mnt/extra-addons ; \
+    test ! -d /var/lib/odoo && mkdir -p /var/lib/odoo ; \
+    chown -R odoo /var/lib/odoo
```

Comment:
Why not just have:
```
&& chown -R odoo /var/lib/odoo \
&& chown -R odoo /mnt/extra-addons
```
That is what I am using (without checking for the existence of the directory first, like you're doing in line 43) .. and it works just fine (and resolves 404 issues when I add all the enterprise modules)

Also there should be no need to check if the dir exists because according to Docker documentation ["Volumes are initialized when a container is created"](https://docs.docker.com/engine/tutorials/dockervolumes/#data-volumes)

When you build the image docker creates intermediary containers at which point the volume would be initialized.